app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java

```diff
@@ -57,6 +57,7 @@
 
     private MainActivityBinding binding;
     private MainNavHeaderBinding headerBinding;
+    private static Fragment fragment;
```

Comment:
Never store fragments or context statically unless it's application context

Reply:
Alright

Reply:
Also address the issues raised by codacy, I thought you were working on those

Reply:
Yes the issues are raised probably because the app is crashing, once the app becomes stable I will look into it

Reply:
No the app crash doesn't reflect in codacy reviews. It means that the code written has some stylistic or semantic mistakes

Reply:
Alright I will resolve the issues raised by codacy first